test/fail_compilation/fail19890a.d

```diff
@@ -0,0 +1,7 @@
+// PERMUTE_ARGS:
+/*
+---
+fail_compilation/fail19890a.d(8): Error: `void[/^[0-9]+(LU)?$/]` size 1 * /^[0-9]+$/ exceeds 0x7fffffff size limit for static array
```

Comment:
It would be nice if the error message could mention that a negative value has been used.

Reply:
That'll be a feature request for master.